app/views/dashboards/_subject.html.erb

```diff
@@ -23,6 +23,17 @@
   </div>
 <% end %>
 
+<% if user.homepage_url.present? %>
+  <div class="flex items-center mb-4 text-b3 lg:text-b2">
+    <%= icon_tag("link", color: :primary, class: "w-6 text-orange mr-3") %>
+    <p class="text-neutral-800 dark:text-white"><%=
+      link_to(
```

Comment:
note to self: suggest the appropriate `rel=none; noreferrer` options

Reply:
looks like `rel="nofollow"` is what github uses on their profile links. I'm agnostic about "noreferrer" in addition.